src/main/webapp/onelive.html

```diff
@@ -28,6 +28,230 @@
 
 <main>
     <!--main content goes here-->
+    <section class="section">
+        <div class="container">
+            <div class="container mt-md-5 d-none d-md-block" style="position: absolute; z-index: 1000">
+                <h1 class="mt-md-5 mb-md-5" style="font-size: 3.3rem; line-height: 1.5;"><span class="font-weight-700">Meet the</span>
+                    Real Life Heroes</h1>
+                <h1 class="mb-0 text-darker font-weight-lighter text-uppercase">one <span
+                        class="font-weight-bold">Live</span></h1>
+                <p class="text-darker">Lorem ipsum dolor sit amet</p>
```

Comment:
Remove dummy text

Reply:
I commented it so if someone needs to put a slogan there we can uncomment it and replace the dummy text. (There was a space to put a slogan in the design but none provided that why I kept it.)